### ISSUE TITLE
toc styles

### ISSUE THREAD

**Issue by habamax** [asciidoctor/asciidoctor-pdf#621]
1. in the documentation there is `toc_dot_leader` with `font_color` example. Although default theme has different name: `dot_leader_color: dddddd`

2. you can actually style `toc_h2_font_style` to be bold, but dotleader then overlaps things
![asciidoctor-pdf-toc](https://cloud.githubusercontent.com/assets/234774/19264641/406f354e-8faa-11e6-925a-f499141f70d4.png)

3. Is there a way (or may be plans) to be able to remove leader dots from certain toc levels (h2)?

**Comment by mojavelinux** [comment]
Looks like a forgot to update the name of the key for the dot leader font color in the default theme. I just applied a fix.

**Comment by mojavelinux** [comment]
> Is there a way (or may be plans) to be able to remove leader dots from certain toc levels (h2)?

No plans at the moment. If you can propose how that might be configured in the theme, that will help move it forward.

Perhaps?

````
toc:
  h2_dot_leader_display: none
```

**Comment by mojavelinux** [comment]
I submitted a fix that corrects the dot calculation when a font style is applied to the section title.

**Comment by mojavelinux** [comment]
Another idea for disabling the dot leader is to set the depth, since usually they are disabled after a certain level.

```
toc:
  dot_leader_levels: 1
```

(Similar in spirit to the toclevels).

That's a bit more vague, but does save typing.

**Comment by habamax** [comment]
I would go for a 

```
toc:
  h2_dot_leader_display: none
```

because I actually don't want to disable them after a certain level, but before. Or simply just for a top-level sections.

**Comment by mojavelinux** [comment]
Thanks for that clarification, @habamax. That's exactly the insight I was looking for. I've opened another issue to track this part of the request. See https://github.com/asciidoctor/asciidoctor-pdf/issues/631.

**Comment by habamax** [comment]
I have used pandoc and latex to generate docs. In latex default toc is actually what I would like to replicate (bold top-level sections/chapters/parts without dots and normal nested sections with dots)

**Comment by habamax** [comment]
Correct number of dots for bold style is confirmed

**Comment by mojavelinux** [comment]
> what I would like to replicate (bold top-level sections/chapters/parts without dots and normal nested sections with dots)

This should now be possible once this is merged.
